### ISSUE TITLE
[Testing] Load Testing Relays Round #1

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/poktroll#20]
## Objective

Complete load test #1.

## Origin Document

The result of the following ticket:
- #229 

Was the following PR:
- #286 

Which was merged here:
- https://dev.poktroll.com/infrastructure/testing/load_testing_plan

## Goals

- Complete load test #1
- Identify bugs & issues in the codebase and infrastructure
- Identify gaps in our tooling & visibility

## Deliverables

- [ ] Document the results (which will be uploaded to [dev.poktroll.com](https://dev.poktroll.com/)) capturing the results of the first load test found [here](https://dev.poktroll.com/infrastructure/testing/load_testing_plan)
- [ ] Document the tools that were added to perform the load test
- [x] Add Grafana visuals for Load Tests.
	* That would unblock drilling down the results ~~by k6 tags, as default functionality does not support that - tags are only available for time series data~~.

## Non-goals / Non-deliverables

- Solving issues or gaps we've identified in the load test
- Items outside the load testing plan

---

**Creator**: @Olshansk
**Co-Owners**: @okdas @red-0ne

**Comment by okdas** [comment]
TODO: create a ticket to add Grafana visuals for Load Tests. That would unblock drilling down the results by k6 tags, as default functionality does not support that - tags are only available for time series data.

**Comment by Olshansk** [comment]
@okdas Thank you for capturing that action item. I've started adding deliverables to avoid having too many "small tasks" for our one load testing operation. @red-0ne please do the same as things come to mind!

**Comment by Olshansk** [comment]
@okdas @red-0ne When we go through [1], we should reference [2] put together by @RawthiL. It is a production version of the demand side scalability in practice on Morse.

[1] https://dev.poktroll.com/infrastructure/testing/load_testing_plan
[2] https://forum.pokt.network/t/block-sizes-claims-and-proofs-in-the-multi-gateway-era/5060

**Comment by bryanchriswhite** [comment]
One of the outstanding deliverables is captured in #513, along with additional deliverables. I've moved the other outstanding deliverable ("Document the tools that were added to perform the load test") to #513 as well and am closing this issue for now.
